### ISSUE TITLE
Model is not saved using model.save("model.h5") 

### ISSUE THREAD

**Issue by stylishsam** [broadinstitute/keras-rcnn#204]

**Comment by 0x00b1** [comment]
Hi, @stylishsam! I don’t know if `model.save` works with custom layers. I’ll look into this.

Have you tried using `to_json` and `save_weights`?

**Comment by shobhitpuri** [comment]
I agree. `model.save` doesn't seem to work. Neither does `to_json`. It was giving some error like `TypeError: can't pickle _thread.lock objects`.

**Comment by CameronVetter** [comment]
I just ran into this too.  Seems strange that a bug this severe remains this long.  Is this project abandoned?

Does anyone have a workaround?
